include/itkButterworthBandpass1DFilterFunction.h

```diff
@@ -54,15 +54,14 @@ class ButterworthBandpass1DFilterFunction : public FrequencyDomain1DFilterFuncti
     return x * y;
   };
 
-  itkSetMacro(UpperFrequency, double) itkGetMacro(UpperFrequency, double)
+  itkSetMacro(UpperFrequency, double) itkGetMacro(UpperFrequency, double);
```

Comment:
```suggestion
  itkSetMacro(UpperFrequency, double);
  itkGetMacro(UpperFrequency, double);
```

Reply:
same for other instances

Reply:
Thank you.  I will force-push with these changes.